lib/phonegap/cordova.js

```diff
@@ -262,7 +268,9 @@ CordovaCommand.prototype.addPhoneGapJS = function(options, callback) {
     var self = this;
 
     var match = options.cmd.match(/(prepare|compile|build|run|emulate)[ ]*([\w ]*)/);
+
     if (match) {
+        console.log(match);
```

Comment:
Is this `console.log` supposed to be here? If so, we should use the proper method: `self.phonegap.emit('log', match);`